cylc/flow/scripts/report_timings.py

```diff
@@ -243,10 +243,18 @@ def write_category(self, buf, category, df_reshape, df_describe):
         pass
 
     def _check_imports(self):
+        if sys.version_info > (3, 11):
+            raise CylcError(
+                'Cylc Report Timings is deprecated: It is incompatible'
```

Comment:
Deprecated means it is still available, which it isn't

Reply:
No, it is only unavailable if you use Python 3.12: If you use an older Python it's fine.

Reply:
This error will only be raised on 3.12+ at which point it's too late to say "deprecated". We can warn about it being deprecated on lower Python versions

```suggestion
                'Cylc Report Timings is incompatible'
```